s3/Acornfile

```diff
@@ -48,68 +50,94 @@ services: "admin": {
 	}]
 }
 
-jobs: apply: {
-	build: {
-		context:    "."
-		dockerfile: "Dockerfile"
-		additionalContexts: {
-			common: "../libs"
-			utils:  "../utils"
+jobs: {
+	cleanup: {
+		build: {
+			context: "../utils/cleanup/"
+			dockerfile: "../utils/cleanup/Dockerfile"
 		}
+		env: {
+				ACORN_EXTERNAL_ID:   "@{acorn.externalID}"
+				CDK_RUNNER_DELETE_PROTECTION: "\(args.deletionProtection)"
+		}
+		events: ["delete"]
+		permissions: rules: [{
+				apiGroup: "aws.acorn.io"
+				verbs: [
+					 "cloudformation:DescribeStacks",
+					 "cloudformation:DescribeStackResources",
+					 "s3:ListBucketVersions",
+					 "s3:DeleteObject",
+					 "s3:DeleteObjectVersion"
+				]
+				resources: ["*"]
+	  }]
 	}
-	files: {
-		"/app/config.json":      std.toJSON(args)
-	}
-	env: {
-		CDK_DEFAULT_ACCOUNT: "@{secrets.aws-context.account-id}"
-		CDK_DEFAULT_REGION:  "@{secrets.aws-context.aws-region}"
-		VPC_ID:              "@{secrets.aws-context.vpc-id}"
-		ACORN_ACCOUNT:       "@{acorn.account}"
-		ACORN_NAME:          "@{acorn.name}"
-		ACORN_PROJECT:       "@{acorn.project}"
-		ACORN_EXTERNAL_ID:   "@{acorn.externalID}"
+
+	apply: {
```

Comment:
This should have a `dependsOn: ["cleanup"] ` ??

Reply:
Good point. Added.